Source/IGListBindingSectionController.m

```diff
@@ -73,7 +73,7 @@ - (void)updateAnimated:(BOOL)animated completion:(void (^)(BOOL))completion {
             }
         }];
         
-        
+        [batchContext invalidateLayoutInSectionController:self atIndexes:result.updates];
```

Comment:
This makes me a little nervous as we're now changing some core functionality within Instagram. Can we add a flag to enable this so that our team can test the change's impact on performance and correctness?